### ISSUE TITLE
:follow-hint crashes outside of hint mode

### ISSUE THREAD

**Issue by The-Compiler** [qutebrowser/qutebrowser#1105]
```
Traceback (most recent call last):
  File "/home/florian/proj/qutebrowser/git/qutebrowser/commands/runners.py", line 227, in run_safely
    self.run(text, count)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/commands/runners.py", line 221, in run
    result.cmd.run(self._win_id, args)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/commands/command.py", line 505, in run
    self.handler(*posargs, **kwargs)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/browser/hints.py", line 957, in follow_hint
    if self._context.to_follow is None:
AttributeError: 'NoneType' object has no attribute 'to_follow'
```

**Comment by The-Compiler** [comment]
Still seems to exist in some way on v1.0 and v0.11:

- https://crashes.qutebrowser.org/view/28573343
- https://crashes.qutebrowser.org/view/8633ee31

**Comment by user202729** [comment]
This should also be fixed because the fix to #3892 ensure that `_context` is not None.

In the second crash report above, the cause is the same (switching tab before `_start_cb` is run)

**Comment by The-Compiler** [comment]
Good catch - agreed, this probably is fixed!
